doc/index.md

```diff
@@ -0,0 +1,39 @@
+# Building an EAP server for the cloud
+
+In order to provision an EAP server for the cloud, you have first to integrate the [EAP Maven plugin](https://github.com/jbossas/eap-maven-plugin/)
+into the pom.xml file of your project.
+
+The "package" goal of the plugin allows you to:
+
+* Provision an EAP server tailored to your needs.
+* Incorporate custom content (keystores, properties files, ...).
+* Execute CLI scripts.
+* Deploy your application.
+
+The generated server is then ready to be installed inside a Docker Image to be deployed in your cluster.
+
+# Provisioning with the cloud feature-pack
+
+The cloud feature-pack is to be provisioned along with the EAP Galleon feature-pack. This is configured in the ``feature-packs`` configuration option 
+of the EAP Maven plugin.
+
+For example:
+
+```xml
+<feature-packs>
+  <feature-pack>
+    <location>org.jboss.eap:wildfly-ee-galleon-pack</location>
+  </feature-pack>
+  <feature-pack>
+    <location>org.jboss.eap.cloud:eap-cloud-galleon-pack</location>
```

Comment:
names... for my own understanding and for future references, we will have 2 duos of feature packs:

* EAP 8
  * org.jboss.eap:wildfly-ee-galleon-pack
  * org.jboss.eap.cloud:eap-cloud-galleon-pack
* EAP XP
  * org.jboss.eap:wildfly-galleon-pack
  * org.jboss.eap.cloud:eap-xp-cloud-galleon-pack

Is that correct?

Reply:
Correct.